.travis.yml

```diff
@@ -4,4 +4,4 @@ sudo: required
 dist: trusty
 before_install: ./maintainers/scripts/travis-nox-review-pr.sh nix
 install: ./maintainers/scripts/travis-nox-review-pr.sh nox
-script: ./maintainers/scripts/travis-nox-review-pr.sh build
+script: travis_wait 119 ./maintainers/scripts/travis-nox-review-pr.sh build
```

Comment:
Why would you expect that print to work? Because it's likely to fall inside the polling interval? It doesn't look travis has any way of knowing that it is from inside `travis_wait` or that it would care.